pkg/cmd/pr/edit/edit.go

```diff
@@ -174,16 +174,34 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 }
 
 func editRun(opts *EditOptions) error {
+	editable := opts.Editable
+	lookupFields := []string{"id", "url", "title", "body", "baseRefName"}
+
+	if opts.Interactive {
+		lookupFields = append(lookupFields, "assignees", "labels")
+	}
+
+	if opts.Interactive || editable.Reviewers.Edited && len(editable.Reviewers.Remove) != 0 {
+		lookupFields = append(lookupFields, "reviewRequests")
+	}
+
+	if opts.Interactive || editable.Projects.Edited {
+		lookupFields = append(lookupFields, "projectCards")
+	}
```

Comment:
Should v2 projects be included in this here?

```suggestion
	if opts.Interactive || editable.Projects.Edited {
		lookupFields = append(lookupFields, "projectCards", "projectItems")
	}
```

Reply:
was excluded in the original pr, I assume it wasn't on purpose. we need the v2 projects idd